apps/external-devices/device-descriptors/PhaseSpace_Motion_Capture.json

```diff
@@ -0,0 +1,19 @@
+{
+    "deviceVendor": "PhaseSpace",
+    "deviceName": "Motion Capture",
+    "author": "Kevin M. Godby <kevin@godby.org>",
+    "version": 1,
+    "lastModified": "2016-12-06T22:17:44Z",
+    "interfaces": {
+        "analog": {
```

Comment:
Unless this actually reports analogs and buttons (which source inspection suggests it does not), remove the analog and button sections - their presence is enough to trigger path creation.